src/core/FilePatternDescriptor.ts

```diff
@@ -0,0 +1,7 @@
+
+interface FilePatternDescriptor {
+  pattern: string;
+  included: boolean;
+}
+
+export default FilePatternDescriptor;
```

Comment:
I think this should be called `InputFileDescriptor` instead if *FilePatternDescriptor*.